packages/booter-rest/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@loopback/booter-rest",
```

Comment:
If we keep this extension, then I'd like us to rename it to `@loopback/model-api-booter`. I believe the implementation can support any API style, not only REST.

Having said that, the booter is very simple and pretty un-opinionated, I think we should add it to core `@loopback/boot` package. All opinionated parts are handled by model-api-builders like `@loopback/rest-crud`.